e2etest/get_stat_value_test.go

```diff
@@ -48,42 +48,56 @@ func TestGetStatValue(t *testing.T) {
 		statVar    string
 		place      string
 		goldenFile string
+		mmethod    string
 		wantErr    bool
 	}{
 		{
 			"Count_Person",
 			"country/USA",
 			"count_person.json",
+			"",
 			false,
 		},
 		{
 			"Count_CriminalActivities_CombinedCrime",
 			"geoId/06",
 			"total_crimes.json",
+			"",
 			false,
 		},
 		{
 			"Median_Age_Person",
 			"geoId/0649670",
 			"median_age.json",
+			"",
 			false,
 		},
 		{
 			"Amount_EconomicActivity_GrossNationalIncome_PurchasingPowerParity_PerCapita",
```

Comment:
Should we eventually add other obs properties to test?

Reply:
yes. will can enrich the tests later.